events/2024.11.Munich/Readme.md

```diff
@@ -13,6 +13,7 @@ Stakeholders from related SDOs are welcome to contribute.
 |--------------|-----------------------------------------------|----------|--------|---------|--------------------------------------------------------------------|-----------------------------------------------------|-------|
 | Siemens      | Sentron PAC (energy meter)                    |  x       |        |         | LAN                                                                | Modbus                                              |   tbc |
 | Siemens      | Industrial Edge                               |  x       |        |         | LAN, connection to the Internet                                    | OPC UA, HTTP                                        |   tbc |
+| Siemens      | Microscope (Openflexure)                      |  x       |        |         | LAN                                                                |                                                     |   tbc |
 | TU Munich    | Hololens                                      |  x       |        |         | Wifi                                                               | no sec should be used; QR code to TD is needed      |   tbc |
 | TU Munich    | Robot arm                                     |  x       |        |         | 2m high, 4m width; LAN/Wifi (no Internet); should be protoced      |                                                     |   tbc |
```

Comment:
I realize this is distinct from your submission... `protoced`? Maybe `protected`?
```suggestion
| TU Munich    | Robot arm                                     |  x       |        |         | 2m high, 4m width; LAN/Wifi (no Internet); should be protected      |                                                     |   tbc |
```